app/scenes/more_dms/more_dms_title.js

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React, {PropTypes} from 'react';
+import {connect} from 'react-redux';
+import {
+    View
+} from 'react-native';
+
+import {getTheme} from 'service/selectors/entities/preferences';
+
+import FormattedText from 'app/components/formatted_text';
+
+function MoreDirectChannelsTitle(props) {
```

Comment:
I don't think you actually need this component. This is just repeating what is already in the ```navigationSceneConnect``` for the default title component. Since you have the title set in the navigationProps for the scene in ```navigation/routes.js``` there isn't a need to create a title component for this scene. The```navigationSceneConnect``` will take care of rendering the title using the default title component.